packages/lab/stories/tabstrip-next/tabstrip-next.stories.tsx

```diff
@@ -0,0 +1,223 @@
+  width = 600,
+  ...tabstripProps
+}) => {
+  const tabs = ["Home", "Transactions", "Loans", "Checks", "Liquidity"];
+
+  return (
+    <div style={{ width, minWidth: 0, maxWidth: "100%" }}>
+      <TabstripNext defaultActiveTabIndex={0} {...tabstripProps} align="center">
+        {tabs.map((label) => (
+          <TabNext label={label} key={label}>
+            {label}
+          </TabNext>
+        ))}
+      </TabstripNext>
+    </div>
+  );
+};
+
+export const ControlledTabstrip: TabstripStory = ({
+  width = 600,
+  ...tabstripProps
+}) => {
+  const tabs = ["Home", "Transactions", "Loans", "Checks", "Liquidity"];
+  const [activeTabIndex, setActiveTabIndex] = useState<number | undefined>(0);
+
+  return (
+    <div style={{ width, minWidth: 0, maxWidth: "100%" }}>
+      <TabstripNext
+        {...tabstripProps}
+        activeTabIndex={activeTabIndex}
+        onActiveChange={setActiveTabIndex}
+      >
+        {tabs.map((label) => (
+          <TabNext label={label} key={label}>
+            {label}
+          </TabNext>
+        ))}
+      </TabstripNext>
+    </div>
+  );
+};
+
+export const AddTabTabstrip: TabstripStory = ({
+  width = 600,
+  ...tabstripProps
+}) => {
+  const [tabs, setTabs] = useState([
+    "Home",
+    "Transactions",
+    "Loans",
+    "Checks",
+    "Liquidity",
+  ]);
+  const [activeTabIndex, setActiveTabIndex] = useState<number | undefined>(0);
+  const handleAddTab = () => {
+    setTabs((t) => [...t, `Tab ${t.length + 1}`]);
+  };
+
+  return (
+    <div style={{ width, minWidth: 0, maxWidth: "100%" }}>
+      <FlexLayout align="center" gap={1}>
+        <TabstripNext
+          {...tabstripProps}
+          activeTabIndex={activeTabIndex}
+          onActiveChange={setActiveTabIndex}
+        >
+          {tabs.map((label) => (
+            <TabNext label={label} key={label}>
+              {label}
+            </TabNext>
+          ))}
+        </TabstripNext>
+        <Button onClick={handleAddTab}>
+          <AddIcon />
+        </Button>
+      </FlexLayout>
+    </div>
+  );
+};
+
+export const LotsOfTabsTabstrip: TabstripStory = ({
+  width = 600,
+  ...tabstripProps
+}) => {
+  const [tabs] = useState([
+    "Home",
+    "Transactions",
+    "Loans",
+    "Checks",
+    "Liquidity",
+    "With",
+    "Lots",
+    "More",
+    "Additional",
+    "Tabs",
+    "Added",
+    "In order to",
+    "Showcase overflow",
+    "Menu",
+    "On",
+    "Larger",
+    "Screens",
+  ]);
+  const [activeTabIndex, setActiveTabIndex] = useState<number | undefined>(0);
+  return (
+    <div style={{ width, minWidth: 0, maxWidth: "100%" }}>
+      <TabstripNext
+        {...tabstripProps}
+        activeTabIndex={activeTabIndex}
+        onActiveChange={setActiveTabIndex}
+      >
+        {tabs.map((label) => (
+          <TabNext label={label} key={label}>
+            {label}
+          </TabNext>
+        ))}
+      </TabstripNext>
+    </div>
+  );
+};
+
+export const ActiveIndexNull: TabstripStory = ({
```

Comment:
what is the usecase of having active index set to null?

Reply:
Sometimes, when using the tabstrip alone you want no tabs selected by default. You do that with null, since undefined will offload the state to useControlled.

Reply:
I see, makes sense... i think currently that is making the tabs inaccessibly via keyboard in the example, could it be missing a tabindex or something like that?

Reply:
Ah, good catch! I think the first one should be focusable in that case 🤔